test/ShallowWrapper-spec.js

```diff
@@ -1369,6 +1369,328 @@ describe('shallow', () => {
 
     });
 
+    it('should return the defaultProps object', () => {
+      class Foo extends React.Component {
+        render() {
+          return (
+            <div id={this.props.foo}>
+              <div className={this.props.bar}></div>
+            </div>
+          );
+        }
+      }
+
+      Foo.defaultProps = {
+        foo: 'foo',
+        bar: 'bar',
+      };
+
+      const wrapper = shallow(<Foo />);
+      expect(wrapper.props().foo).to.equal('foo');
```

Comment:
@aweary can we close this PR? This test doesn't seem to make sense, `wrapper.props()` must be `{id: 'foo'}` because the root node in `shallow` is the rendered node